src/lightning_app/core/work.py

```diff
@@ -80,6 +81,8 @@ def __init__(
             local_build_config: The local BuildConfig isn't used until Lightning supports DockerRuntime.
             cloud_build_config: The cloud BuildConfig enables user to easily configure machine before running this work.
             run_once: Deprecated in favor of cache_calls. This will be removed soon.
+            start_with_flow: Whether the work should be started at the same time as the Root Flow. Only applies to works
```

Comment:
```suggestion
            start_with_flow: Whether the work should be started at the same time as the root flow. Only applies to works
```
or was the capitalization intentional?